controls/sshd_spec.rb

```diff
@@ -77,7 +77,6 @@
     it { should exist }
     it { should be_file }
     it { should be_owned_by 'root' }
-    it { should be_grouped_into 'root' }
```

Comment:
@techraf yes, you can do in [travis](https://travis-ci.org) in the settings of your fork:

![untitled](https://cloud.githubusercontent.com/assets/1645670/22023674/669767e0-dcc7-11e6-827f-08d064c45054.png)